sdks/python/apache_beam/transforms/timeutil.py

```diff
@@ -115,8 +115,8 @@ def __mod__(self, other):
     return Duration(micros=self.micros % other.micros)
 
 
-MIN_TIMESTAMP = Timestamp(micros=-sys.maxint - 1)
-MAX_TIMESTAMP = Timestamp(micros=sys.maxint)
+MIN_TIMESTAMP = Timestamp(micros=-0x7fffffffffffffff - 1)
```

Comment:
dumb question, but in Python can't you just do `-0x8000000000000000`?  [number of zeros not carefully checked]